cmd/dep/testdata/harness_tests/init/glide/direct-trans-conflict/README.md

```diff
@@ -0,0 +1 @@
+Take a direct dependency on a transient dependency where the versions are conflicted. Resolving should fail.
```

Comment:
This test does not fail since the existing code only removes transient dependencies read from importers. Since there is also a direct dependency here, the test passes.